TreeView/TreeViewAsync.razor.cs

```diff
@@ -21,6 +23,11 @@ public partial class TreeViewAsync<T> : TreeViewBase<T>
             builder.CloseComponent();
         };
 
+        public async Task Refresh()
+        {
+            await RootNode.Refresh();

```

Comment:
Two questions here:

1. Should we name the method `RefreshAsync`?
2. Why do we refresh the tree view on the `RootNode`? What if that's the thing that changed?

Reply:
1. Refresh async is better
2. We refresh the `RootNode` because it is the thing that changed. Children can be updated via `GetChildrenAsync` but so far `GetChildrenAsync(null)` (to get the root nodes) can only be called once on initialization. At first I wanted to refresh the treeview itself but I di not find how